e2e-test/cypress/plugins/index.js

```diff
@@ -115,4 +116,6 @@ module.exports = (on, _config) => {
       return options;
     }
   });
+  _config = dotenvPlugin(_config);
```

Comment:
Rename the _config  to config since it's used now.

Reply:
updated.